docs/guide/common_addons.md

```diff
@@ -71,4 +71,157 @@ dashboard accessible from the outside via `https` on port `443`.
 
 ![FIP](https://raw.githubusercontent.com/sapcc/kubernikus/master/assets/images/docs/containers/kubernetes/loadbalancer2.png)
 
-~> This setup exposes a unauthenticated Dashboard with full access to the cluster. This is a security risk. See the [Access Control](https://github.com/kubernetes/dashboard/wiki/Access-control) documentation for more info.
+~> This setup exposes an unauthenticated Dashboard with full access to the cluster. This is a security risk. See the [Access Control](https://github.com/kubernetes/dashboard/wiki/Access-control) documentation for more info.
+
+## Deploy HANA Express database on Kubernikus
+
+Create a Kubernetes cluster and deploy SAP HANA, express edition containers (database server only).
+
+### Step 1: Create Kubernetes Cluster
+Login to the Converged Cloud Dashboard and navigate to your project. Open `Containers > Kubernetes`. Click `Create Cluster`, choose a cluster name (max. 20 digits), give your nodepool a name, choose a number of nodes and use at least a `m1.large` flavor which offers you `4 vCPU, ~8 GB RAM` per node. Create the `kluster` (Cluster by Kubernikus). 
+
+### Step 2: Connect to your kluster
+Use the following instructions to get access to your Kubernetes Cluster. [Authenticating with Kubernetes](https://kubernikus.eu-nl-1.cloud.sap/docs/guide/authentication/#authenticating-with-kubernetes).
+
+### Step 3: Create the deployments configuration files
```

Comment:
Is this really required? I don't see the secret references anywhere...

Reply:
It's required for the hana express image on docker store (Line 183). Without the secret it will throw a ImagePullBackOff Error.